.github/integration/sda/aai-mock/userinfos/dummy-user.yaml

```
@@ -2,7 +2,7 @@ web-display-name: "Dummy User - submitter"
 
 sub: "test@dummy.org"
 
-name: "Dummy User"
+name: "test@dummy.org"
```

```suggestion
name: "Dummy User"
```

This should be as it is.

---

reverted the commit ea0908da